### PR TITLE
#416 - remove typehint in add_s3_signed_params_to_attachment_image_src

### PR DESCRIPTION
**Pull request by Koli14**
This PR fix this issue: https://github.com/humanmade/S3-Uploads/issues/416

### CODE REVIEW

**Comment by joehoyle** [comment]
I believe we fixed this in https://github.com/humanmade/S3-Uploads/pull/401, which contains a bunch of refactoring. Just waiting on a review from @rmccue to push that forward.

**Comment by Koli14** [comment]
> I believe we fixed this in #401, which contains a bunch of refactoring. Just waiting on a review from @rmccue to push that forward.

I don't see, if it's changed: https://github.com/humanmade/S3-Uploads/blob/dce05fbaf1515a849e905afa604620d22623acb6/inc/class-plugin.php#L583
Although it's still might be fixed...

**Comment by joehoyle** [comment]
Ahh ok good point, I thought that code had been refactored too, but perhaps not.

File: inc/class-s3-uploads.php [hunk 1]
```diff
@@ -483,7 +483,7 @@ public function add_s3_signed_params_to_attachment_url( string $url, int $post_i
 	 * @param integer $post_id
```
Comment (joehoyle) [inline inc/class-s3-uploads.php:483]: Could the phpdoc be updated too to reflect the possible types?

**Comment by joehoyle** [comment]
Hey @kodie I closed this in favour of https://github.com/humanmade/S3-Uploads/pull/474, as there was a big amount of code churn from #401

**Pull request closed without merging**